test_scripts/API/Navigation/UnsubscribeWayPoints/004_UnsubscribeWayPoints_UNSUPPORTED_RESOURCE_Navi_interface_is_not_avaliable.lua

```diff
@@ -0,0 +1,56 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
@dboltovskyi Please edit test case name. It contains error.

Reply:
@AByzhynar Accepted